igvc_gazebo/nodes/sim_color_detector/sim_color_detector.hpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * Node responsible for (color) detecting the lines and barrels in the Gazebo
+ * simulator. Receives the raw image and publishes two different B&W images
+ * (one for line detection, one for barrel detection) that each highlight
+ * their respective features in white.
+ */
+
+#include <cv_bridge/cv_bridge.h>
+#include <ros/ros.h>
+#include <sensor_msgs/Image.h>
+#include <sensor_msgs/image_encodings.h>
+#include <fstream>
+#include <iostream>
```

Comment:
I don't think we need these imports given that we don't do any file operations and all the prints are done using ROS.